dlc_developer_config.toml

```diff
@@ -22,8 +22,8 @@ benchmark_mode = false
 
 [build]
 # Frameworks for which you want to disable both builds and tests
-# available frameworks - ["autogluon", "huggingface_tensorflow", "huggingface_pytorch", "huggingface_tensorflow_trcomp", "huggingface_pytorch_trcomp", "tensorflow", "mxnet", "pytorch"]
-skip_frameworks = []
+# available frameworks - ["autogluon", "huggingface_tensorflow", "huggingface_pytorch", "tensorflow", "mxnet", "pytorch"]
+skip_frameworks = ["huggingface_tensorflow", "huggingface_pytorch", "tensorflow", "mxnet", "pytorch"]
```

Comment:
Just curious, why are we now skipping these frameworks?

Reply:
To reduce build costs. Once all the framework tests are passed, changes to this file are reverted and full test suite  runs.